cardscan-base/src/main/java/com/getbouncer/cardscan/base/SSDOcrDetect.java

```diff
@@ -36,20 +39,20 @@ static boolean isInit() {
     private String ssdOutputToPredictions(Bitmap image){
         ArrUtils arrUtils = new ArrUtils();
 
-        float[][] k_boxes = arrUtils.rearrangeOCRArray(ssdOcrModel.outputLocations, ssdOcrModel.featureMapSizes,
-                ssdOcrModel.NUM_OF_PRIORS_PER_ACTIVATION, ssdOcrModel.NUM_OF_CORDINATES);
-        k_boxes = arrUtils.reshape(k_boxes, ssdOcrModel.NUM_OF_PRIORS, ssdOcrModel.NUM_OF_CORDINATES);
+        float[][] k_boxes = arrUtils.rearrangeOCRArray(ssdOcrModel.outputLocations, SSDOcrModel.featureMapSizes,
+                SSDOcrModel.NUM_OF_PRIORS_PER_ACTIVATION, SSDOcrModel.NUM_OF_CORDINATES);
```

Comment:
nit: not introduced in this PR, but typo in `CORDINATES`

Reply:
may as well fix this :)